### PR TITLE
About tab carousel

### PR DESCRIPTION
**Pull request by maurizi**
## Overview

Adds a stubbed out version of the About tab, bringing in a carousel library but no content

### Demo

localhost_3474_

### Notes

I tried to use nesting in my CSS, which `styled-components` [says it can do](https://www.styled-components.com/docs/faqs#can-i-nest-rules), but it didn't seem to work.

## Testing Instructions

 * `scripts/update`
 * `scripts/server`

### CODE REVIEW

File: src/app/src/reducers.js [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
+import { changeAboutSlide, hideScreensaver, showScreensaver } from './actions';
 
 export const initialState = {
+    aboutSlideIndex: 0,
     isScreensaverVisible: true,
 };
 
 export default createReducer(
     {
+        [changeAboutSlide]: (state, payload) =>
```
Comment (fungjj92) [inline src/app/src/reducers.js]: Nice job adding this action & reducer set! I think we can tweak the name to give its function some clarity. What do you think about `setAboutCarouselIndex` or `saveAboutSlideIndex` or similar?

File: src/app/package.json [hunk 2]
```diff
@@ -44,6 +44,7 @@
     "jest-watch-typeahead": "^0.2.1",
     "leaflet": "~1.3.1",
     "mini-css-extract-plugin": "0.5.0",
+    "nuka-carousel": "^4.5.4",
```
Comment (fungjj92) [inline src/app/package.json]: I led by poor example; let's pin our package versions to at least minor versions `~`

File: src/app/src/css/about.css [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+.about__slide {
+    height: 80vh;
+    padding: 1rem;
+}
+
+/* nuka-carousel marks previous slides as aria-hidden
+   This means we need to override our aria-hidden style so we
+   can still see slivers of the surrounding slides */
+.about__slide[aria-hidden="true"] {
```
Comment (fungjj92) [inline src/app/src/css/about.css]: The slides being hidden is actually a cause of CSS in `navbar.css` that hides all elements where `aria-hidden="true"`, https://github.com/azavea/pwd-fishway/blob/develop/src/app/src/css/navbar.css#L8

I also couldn't figure out nested CSS for whatever reason which would prevent this run-in 🤔 We can update the comment in this `about.css` and leave this CSS rule or adjust the `navbar.css` while getting rid of this CSS rule.

**Review by fungjj92 — CHANGES_REQUESTED** [review]
Nice first React PR! Works well, and the carousel is charming. Can't wait to demo next week. Some clean up comments.

File: src/app/package.json (hunk 2 above)
Comment (maurizi) [reply src/app/package.json]: Ok. FYI this was just the default given by `yarn add nuka-carousel`, but I switched it regardless.

File: src/app/src/css/about.css (hunk 3 above)
Comment (maurizi) [reply src/app/src/css/about.css]: Went with option 2 and scoped the selector in `navbar.css` to only apply to the tab panes.

**Review by fungjj92 — APPROVED** [review]
Nice work!

**Pull request merged**